frontend/src/components/SubmissionWidget.tsx

```diff
@@ -114,15 +111,25 @@ const SubmissionWidget: React.FC<SubmissionWidgetProps> = ({
               </h4>
               {globalDocs.map((doc) => (
                 <div key={doc.title} className="mb-4">
-                  <ul>
-                    <Link
-                      to={generateDocRoute(doc.title, true)}
-                      target="_blank" // This is what is causing the issue.
-                      rel="noopener noreferrer"
-                      className="underline text-blue-600 hover:text-blue-800"
-                    >
-                      {doc.title}
-                    </Link>
+                  <ul className="list-disc pl-6 text-black">
+                    <li>
+                      <button
+                        onClick={() => openDocInNewTab(doc)}
+                        className="text-blue-500 hover:text-blue-300"
+                      >
+                        {doc.title}
+                      </button>
+                      <li>
+                        <Link
+                          to="../../public/python-3.13-docs-html/index.html"
```

Comment:
As written, this code either does not work (the link needs to be outside of the globalDocs.map), or my mental model of how the docs are served is not in line with yours. Either way, you need to update the Readme/Documentation and possible fix the error.

Reply:
I moved the link outside of the globalDocs.map. Is this what you were looking for?

```jsx
{docsTab === "global" && (
          <div>
            <h4>
              <strong>Global Documentation</strong>
            </h4>
            <div className="mb-4">
              <ul className="list-disc pl-6 text-black">
                {globalDocs.map((doc) => (
                  <li key={doc.title}>
                    <button
                      onClick={() => openDocInNewTab(doc)}
                      className="text-blue-500 hover:text-blue-300"
                    >
                      {doc.title}
                    </button>
                  </li>
                ))}
                <li>
                  <Link
                    to="/python-3.13-docs-html/index.html"
                    target="__blank"
                    rel="noopener noreferrer"
                    className="text-blue-500 hover:text-blue-300"
                  >
                    Python 3 Documentation
                  </Link>
                </li>
              </ul>
            </div>
          </div>
        )}
```

Reply:
Good! I like it!!